ToolBox/LeanDataWriter.cs

```diff
@@ -59,9 +59,10 @@ public LeanDataWriter(Resolution resolution, Symbol symbol, string dataDirectory
             {
                 _dataType = TickType.Quote;
             }
-
+            
             // Can only process Fx and equity for now
-            if (_securityType != SecurityType.Equity && _securityType != SecurityType.Forex && _securityType != SecurityType.Cfd)
+            if (_securityType != SecurityType.Equity && _securityType != SecurityType.Forex && _securityType != SecurityType.Cfd
```

Comment:
I'll implement a writer for this custom data. 

But maybe worth consider making an abstraction from the LDW in order to leverage the actual system of files for minute data.